File: HrNewPortal.Models/Story.cs
using System;

namespace HrNewsPortal.Models
{
    public class Story
    {
        /// <summary>
        /// The item's unique identifier.
        /// </summary>
        public int Id { get; set; }
        
        /// <summary>
        /// The user name of the item's author.
        /// </summary>
        public string By { get; set; }

        /// <summary>
        /// Creation UTC date of the item.
        /// </summary>
        public DateTime? Time { get; set; }

        /// <summary>
        /// The identifiers of the item's
        /// comments, in ranked display order.
        /// </summary>
        public int[] Kids { get; set; }

        /// <summary>
        /// The comments, in ranked display order.
        /// </summary>
        public Comment[] Comments { get; set; }

        /// <summary>
        /// The URL of the story.
        /// </summary>
        public string Url { get; set; }

        /// <summary>
        /// The story's score, or the votes
        /// for a pollopt (poll option).
        /// </summary>
        public int Score { get; set; }
    }
}

File: HrNewsPortal.Portal/ClientApp/src/app/models/story.ts
import { Comment } from "./comment";

export class Story {
  // The item's unique identifier.
  public id: number;

  // The user name of the item's author.
  public by: string;

  // Creation UTC date of the item.
  public time?: Date;

  // The identifiers of the item's
  // comments, in ranked display order.
  public kids: number[];

  // The comments, in ranked display order.
  public comments: Comment[];

  // The URL of the story.
  public url: string;

  // The story's score, or the votes
  // for a pollopt (poll option).
  public score: number;
  
  // The title of the story, poll, or job.
  public title: string;
}
